base_fontawesome/__openerp__.py

```diff
@@ -5,8 +5,8 @@
 {
     'name': 'Base Fontawesome',
     'summary': """Up to date Fontawesome resources.""",
-    'version': '9.0.4.7.0',
-    'license': 'LGPL-3',
+    'version': '8.0.0.1.0',
```

Comment:
The version should be set to `8.0.4.7.0`, the version here is the same as Fontawesome.